activesupport/lib/active_support/json/decoding.rb

```diff
@@ -19,8 +19,8 @@ class << self
       #
       #   ActiveSupport::JSON.decode("{\"team\":\"rails\",\"players\":\"36\"}")
       #   => {"team" => "rails", "players" => "36"}
-      def decode(json)
-        data = ::JSON.parse(json)
+      def decode(json, options = {})
```

Comment:
Depending on how we feel about https://github.com/rails/rails/pull/54172#discussion_r1913903792, could this be replaced with splatted keyword arguments?

```suggestion
      def decode(json, **options)
```